sources/packages/backend/apps/api/src/cra-integration/cra-files/cra-file-footer.ts

```diff
@@ -1,11 +1,7 @@
-import { StringBuilder } from "../../utilities/string-builder";
-import {
-  DATE_FORMAT,
-  SPACE_FILLER,
-  NUMBER_FILLER,
-  TransactionCodes,
-} from "../cra-integration.models";
-import { FixedFormatFileLine } from "../../services/ssh/sftp-integration-base.models";
+import { DATE_FORMAT, NUMBER_FILLER, SPACE_FILLER } from "@sims/integration";
+import { FixedFormatFileLine } from "@sims/integration/services/ssh/sftp-integration-base.models";
+import { StringBuilder } from "@sims/integration/utilities";
```

Comment:
I would say that `StringBuilder` has nothing integration-related and I guess that it is generic enough to be moved to the generic utility library, IMO.

Reply:
but it is only used for file integration, right?

Reply:
moved